pkg/kubecfg/pack.go

```diff
@@ -201,6 +207,20 @@ func (c PackCmd) pushOCIBundle(ctx context.Context, ref string, bodyBlob []byte,
 	return nil
 }
 
+func getSourceRevision(rootFile string) string {
+	repo, err := git.PlainOpenWithOptions(rootFile, &git.PlainOpenOptions{DetectDotGit: true})
+	if err != nil {
+		log.Printf("error opening git repo: %s", err)
```

Comment:
we're not requiring kubecfg to operate in a git repo; so we don't want to annoy people that are not in a git repo.

a debug log is useful though since it may explain why the revision is not detected. 

But perhaps the message could say why this error matters? e.g. "populating vcs revision for OCI annotation: error opening git repo: %s"